README.md

```diff
@@ -271,9 +271,12 @@ For more information, please see https://www.bis.doc.gov
 
 Licensing
 =========
-Docker is licensed under the Apache License, Version 2.0. See
-[LICENSE](https://github.com/docker/docker/blob/master/LICENSE) for the full
-license text.
+Copyright © 2014-2015 Docker, Inc. All rights reserved, except as follows:
+
+* Except where otherwise noted, the code in this repository is licensed under the Apache License, Version 2.0. See [LICENSE](https://github.com/docker/docker/blob/master/LICENSE) for the full license text. 
+
+* The files in the `docs` folder are licensed to end users under the Creative Commons Attribution 4.0 International License under the terms and conditions set forth in the file [`LICENSE-DOCS`](LICENSE-DOCS). You may
+obtain a duplicate copy of the same license, titled CC-BY-SA-4.0, at http://creativecommons.org/licenses/by/4.0/.
```

Comment:
What's the license on the man page dir?

I suspect we'd be better off licensing all markdown / docs files as CC or Apache - as stated, there is no license stated for